### PR TITLE
user table + login service

### PR DESCRIPTION
**Pull request by Teanquisitor**

### CODE REVIEW

File: project/src/main/java/com/carlease/project/application/ApplicationMapper.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.carlease.project.application;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel = "spring")
+public interface ApplicationMapper {
+
+    ApplicationMapper INSTANCE = Mappers.getMapper( ApplicationMapper.class );
+
+    //add fields here
```
Comment (Teanquisitor) [inline project/src/main/java/com/carlease/project/application/ApplicationMapper.java:10]: remove comments

**Review by ingridarev — APPROVED** [review]

File: project/src/main/java/com/carlease/project/application/ApplicationServiceImpl.java [hunk 2]
```diff
@@ -1,16 +1,34 @@
...
 
-    public ApplicationServiceImpl(ApplicationRepository applicationRepository) {
+    private final UserRepository userRepository;
+
+    @Autowired
+    private ApplicationMapper applicationMapper;
+
+    public ApplicationServiceImpl(ApplicationRepository applicationRepository,
+                                  CarRepository carRepository, UserRepository userRepository
+                                  ) {
```
Comment (Teanquisitor) [inline project/src/main/java/com/carlease/project/application/ApplicationServiceImpl.java:27]: bring ") {" to the previous line

File: project/src/main/java/com/carlease/project/application/ApplicationServiceImpl.java [hunk 3]
```diff
@@ -1,16 +1,34 @@
...
         this.applicationRepository = applicationRepository;
+        this.carRepository = carRepository;
+        this.userRepository = userRepository;
+
```
Comment (Teanquisitor) [inline project/src/main/java/com/carlease/project/application/ApplicationServiceImpl.java:31]: remove empty line

File: project/src/main/java/com/carlease/project/car/CarRepository.java [hunk 4]
```diff
@@ -1,9 +1,13 @@
...
 import org.springframework.stereotype.Repository;
+import java.util.List;
 
 @Repository
 public interface CarRepository extends JpaRepository<Car, Long> {
-}
+    List<Car> findByMake(String make);
+    Car findByMakeAndModel(String make, String model);
+
+
```
Comment (Teanquisitor) [inline project/src/main/java/com/carlease/project/car/CarRepository.java:12]: remove empty lines

**Pull request merged**